src/lib/tree/tree.md

```diff
@@ -0,0 +1,50 @@
+The `mat-tree` provides a Material Design styled tree that can be used to display hierarchy
+data.
+
+This tree builds on the foundation of the CDK tree and uses a similar interface for its
+data source input and template, except that its element and attribute selectors will be prefixed 
+with `mat-` instead of `cdk-`.
+
+### Features
+
+The `<mat-tree>` itself only deals with the rendering of a tree structure.
+Additional features can be built on top of the tree by adding behavior inside node templates
+(e.g., padding and trigger). Interactions that affect the
+rendered data (such as expand/collapse) should be propagated through the table's data source.
+
+### TreeControl
+
+The `TreeControl` expand/collapse of the tree node. Users can expand/collapse a tree node
```

Comment:
The first sentence sounds like it's missing something. 
```markdown
The `TreeControl` controls the expand/collapse state of tree nodes.
```

After that, it would be good to explain the two types of `TreeControl` and when they'd be used before going into further detail